### ISSUE TITLE
Make Spectrum A Signed Plug-in

### ISSUE THREAD

**Issue by ingomohr** [AObuchow/Eclipse-Spectrum-Theme#112]
When I install the Spectrum theme to my Eclipse, I'm notified that the plug-in isn't signed and whether I want to "Install anyway".

Maybe we can change that an provide the theme as signed plug-in?

**Comment by AObuchow** [comment]
Yes for sure !! I had briefly looked into that a while back, I was reading https://nirmalsasidharan.wordpress.com/2010/09/04/signing_eclipse_plugins/ on this topic

**Comment by ingomohr** [comment]
Cool! Maybe it's rather easy after all ^^

**Comment by AObuchow** [comment]
If anyone’s interested in working on this feel free to let me know :) I’ll probably only address it after some other topics.

**Comment by ingomohr** [comment]
I would like to check this out. Maybe this is a good alternative in case I'd be stuck in the DevOps stuff. ;)

**Comment by AObuchow** [comment]
@ingomohr for sure it might be a bit more refreshing when you need a break ;) And it’ll be a nice improvement, especially for first time users of Spectrum Theme

**Comment by ingomohr** [comment]
I found another documentation  - which might be a bit more recent:
https://www.vogella.com/tutorials/EclipseTycho/article.html#signing-plug-ins
